packages/core/package.json

```diff
@@ -15,7 +15,6 @@
   "dependencies": {
     "@department-of-veterans-affairs/react-components": "workspace:*",
     "@department-of-veterans-affairs/web-components": "workspace:*",
-    "date-fns-tz": "^1.1.1",
```

Comment:
@bkjohnson you are right there will be build errors on `vets-website` because `component-library` was the package bringing in the `date-fns-tz` peer dependency. Would be interested in your and @k80bowman opinion on the best move here. We are using `date-fns-tz` in all the SIP jsx files per the PR Robin completed last week: https://github.com/department-of-veterans-affairs/vets-website/commit/48b2f509e811112bbd5c5941c81db90693f08c8f

<img width="601" alt="Screen Shot 2022-01-26 at 3 45 15 PM" src="https://user-images.githubusercontent.com/11822533/151244706-54e1785c-cbcb-49c2-a99a-a4d2803e4fbc.png">

This was done so we could move away from using `moment`. Now we also have `date-fns` as a dependency in `vets-website` core `package.json` https://github.com/department-of-veterans-affairs/vets-website/blob/master/package.json#L268

We could potentially do a ticket that fixes the files for SIP in the image above to move away from `date-fns-tz` and then we'd be ok since both `vets-website` and `component-library` would no longer need it but interested to hear your thoughts.

Reply:
If we are using date-fns-tz in vets-website, should it just be added as a dependency there?

Reply:
Yeah, if `vets-website` needs it then it should be added as a dependency there since our components don't use it anymore.

Reply:
@k80bowman @bkjohnson I created a new section on the ticket named: Acceptance Criteria Upon Component Library Merge with the following tasks
* Test IE 11 Support
* Add date-fns-tz as a dependency in vets-website repo package.json